configs/segmenter/segmenter.yml

```diff
@@ -121,5 +121,5 @@ Models:
     Metrics:
       mIoU: 52.16
       mIoU(ms+flip): 53.65
-  Config: ''
-  Weights: ''
+  Config: configs/segmenter/segmenter_vit-l_mask_8x1_640x640_160k_ade20k.py
+  Weights: https://download.openmmlab.com/mmsegmentation/v0.5/segmenter/segmenter_vit-l_mask_8x1_512x512_160k_ade20k/segmenter_vit-l_mask_8x1_512x512_160k_ade20k_20220105_162750-7ef345be.pth
```

Comment:
Shouldn't this URL also be changed? To reflect the resolution of 640x640, it should be something like this:

https://download.openmmlab.com/mmsegmentation/v0.5/segmenter/segmenter_vit-l_mask_8x1_640x640_160k_ade20k/segmenter_vit-l_mask_8x1_640x640_160k_ade20k_20220105_162750-7ef345be.pth

Reply:
Thanks for this kind reminder, we will rename the URL later, and create another pr to fix it.